bundles/action/org.openhab.action.pushover/src/main/java/org/openhab/action/pushover/internal/Pushover.java

```diff
@@ -253,10 +257,14 @@ public static boolean pushover(@ParamDoc(name = "apiKey", text = "Your applicati
 
             if (!StringUtils.isEmpty(device)) {
                 addEncodedParameter(data, MESSAGE_KEY_DEVICE, device);
+            } else if (StringUtils.isEmpty(device) && !StringUtils.isEmpty(defaultDevice)) {
```

Comment:
The part `StringUtils.isEmpty(device) && ` is unneeded and would read better if it were removed.